### DISCUSSION TITLE
Comments still showing on preview builds even though I've disabled them everywhere.

### DISCUSSION THREAD

**Discussion by gurkang** [vercel/community#1901] [Help] [answered]
### Summary

I've disabled comments at the account level and project level. 

<img width="948" alt="Screenshot 2023-03-28 at 23 00 39" src="https://user-images.githubusercontent.com/9166287/228299356-a3eb428c-b4db-4cd6-a530-ced298f0ca53.png">
<img width="981" alt="Screenshot 2023-03-28 at 23 00 55" src="https://user-images.githubusercontent.com/9166287/228299380-3d0e97c8-186c-4ca5-9f3f-85e1e9f861dd.png">

I've even tried adding the `env` variable to shut it off:

<img width="974" alt="Screenshot 2023-03-28 at 23 02 42" src="https://user-images.githubusercontent.com/9166287/228300170-cd5b4ac4-c776-4d32-aad8-805cea319550.png">

I've deployed the preview deployment 3 times.

And yet with all of this, I still see the "login to comment" box in my preview builds.

How can I just force this away. I do not want nor do I need it right now and it's a distraction on the site when testing.

Any help to force this away?

**Comment by amyegan** [comment, marked as answer]
Hi @gurkang. If you have redeployed or created a new preview deployment after disabling comments, please share a deployment URL for us to investigate 🙏

**Reply by gurkang** [reply]
A small update on this. I've noticed the the domain that includes the `git-branch-gurkang-vercel.app` doesn't get the new update for turned off comments. However the randomly generated one `project-name-j3b1xls1x-gurkang.vercel.app` does indeed have the comments turned off by default.

Is there a difference in the way these are deployed?

**Reply by amyegan** [reply]
Thanks for sharing this update. Please share the actual URLs for these examples, or the deployment link or ID, so we can look into the logs and investigate what is going wrong 🙏

**Reply by gurkang** [reply]
@amyegan 
Here's the latest preview url https://cover-mate-12en2w3b6-gurkang.vercel.app/
And the "stable" one where the login button still shows:
https://cover-mate-git-dev-gurkang.vercel.app/

**Reply by amyegan** [reply]
Thanks for sharing these. I don't see the overlay on either one, so this may be caused by your browser cache. Can you try clearing the cache and try again?

**Reply by gurkang** [reply]
It is working now.
I did however at the time of posting try it in several different browsers and even different devices with cleared caches.

At the time the "stable" one was working as intended. However the latest deployment specific URL wasn't working with disabled comments.
It looks like it took quite a bit longer for that one to update.
